web-client/public/js/graph.js

```diff
@@ -1332,7 +1333,7 @@ export var drawGraph = function (network) {
                 return "translate(" + d.x + "," + d.y + ")";
             });
 
-      /* Allows for looping edges.
+            /* Allows for looping edges.
```

Comment:
The reindenting on this comment looks off as well.
```suggestion
      /* Allows for looping edges.
```